README.md

```diff
@@ -1242,6 +1242,17 @@ params do
 end
 ```
 
+You can use the value of one parameter as the default value of some other parameter. In this
+case, if the `primary_color` parameter is not provided, it will have the same value as the `color`
+one. If both of them not provided, both of them will have `blue` value.
```

Comment:
One last thing, I promise that's it after that. We don't wrap text in markdown manually since it shows up, make this all 1-line.

Reply:
Hi @dblock ! Don't worry at all, I really appreciate the reviews :) 

I've just updated my addition to the README file for following 1-line format. However, I did it in that way because all the other sections of the document. 

As you can see here:

https://github.com/ruby-grape/grape/blob/master/README.md?plain=1#L153-L159

If the 1-line is the new convention you want to follow, do you think I can help on moving all the README to 1-line format?

Reply:
Absolutely. Maybe we can find a linter to run in PRs too?

Reply:
Sure! Let me try to PR this :)